coderd/database/dbmem/dbmem.go

```diff
@@ -929,15 +929,20 @@ func (q *FakeQuerier) AcquireNotificationMessages(_ context.Context, arg databas
 		return nil, err
 	}
 
+	// Shift the first "Count" notifications off the slice (FIFO).
+	sz := len(q.notificationMessages)
```

Comment:
why `sz` :) ?

Reply:
`size`

Reply:
I generally agree, although I try to follow this advice: https://go.dev/talks/2014/names.slide#4

> The greater the distance between a name's declaration and its uses, the longer the name should be.

Here it's pretty clear what `sz` does, so brevity is helpful.

Reply:
offtopic: It depends on your background. For me `sz` is reserved for **s**tring **z**ero-terminated (c++).

Reply:
Ha! Fair enough :)